### PR TITLE
Make changelogs more discoverable

### PR DESCRIPTION
**Pull request by stevenroose**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+- [bitcoin CHANGELOG](bitcoin/CHANGELOG.md)
+
+- [addresses CHANGELOG](addresses/CHANGELOG.md)
+- [base58 CHANGELOG](base58/CHANGELOG.md)
+- [hashes CHANGELOG](hashes/CHANGELOG.md)
+- [internals CHANGELOG](internals/CHANGELOG.md)
+- [io CHANGELOG](io/CHANGELOG.md)
+- [primitives CHANGELOG](primitives/CHANGELOG.md)
+- [units CHANGELOG](units/CHANGELOG.md)
```
Comment (storopoli) [inline CHANGELOG.md:12]: Maybe put them in backticks. I find it really useful to find out that things are "code"-related.
```suggestion
- [`bitcoin` CHANGELOG](bitcoin/CHANGELOG.md)
- [`addresses` CHANGELOG](addresses/CHANGELOG.md)
- [`base58` CHANGELOG](base58/CHANGELOG.md)
- [`hashes` CHANGELOG](hashes/CHANGELOG.md)
- [`internals` CHANGELOG](internals/CHANGELOG.md)
- [`io` CHANGELOG](io/CHANGELOG.md)
- [`primitives` CHANGELOG](primitives/CHANGELOG.md)
- [`units` CHANGELOG](units/CHANGELOG.md)
```

File: README.md [hunk 2]
```diff
@@ -203,8 +203,13 @@ Our code is public domain so by all means fork it and go wild :)
 Release notes are done per crate, see:
 
 - [bitcoin CHANGELOG](bitcoin/CHANGELOG.md)
+- [addresses CHANGELOG](addresses/CHANGELOG.md)
+- [base58 CHANGELOG](base58/CHANGELOG.md)
 - [hashes CHANGELOG](hashes/CHANGELOG.md)
 - [internals CHANGELOG](internals/CHANGELOG.md)
+- [io CHANGELOG](io/CHANGELOG.md)
+- [primitives CHANGELOG](primitives/CHANGELOG.md)
+- [units CHANGELOG](units/CHANGELOG.md)
```
Comment (storopoli) [inline README.md:212]: Maybe put them in backticks. I find it really useful to find out that things are "code"-related.
```suggestion
- [`bitcoin` CHANGELOG](bitcoin/CHANGELOG.md)
- [`addresses` CHANGELOG](addresses/CHANGELOG.md)
- [`base58` CHANGELOG](base58/CHANGELOG.md)
- [`hashes` CHANGELOG](hashes/CHANGELOG.md)
- [`internals` CHANGELOG](internals/CHANGELOG.md)
- [`io` CHANGELOG](io/CHANGELOG.md)
- [`primitives` CHANGELOG](primitives/CHANGELOG.md)
- [`units` CHANGELOG](units/CHANGELOG.md)
```

**Review by tcharding — COMMENTED** [review]
Awesome thanks man, this was on my todo list since someone in Nashville also mentioned the changelogs were hard to find.

Concept ACK, but I agree with @storopoli's suggestion.

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+
+A changelog is kept per crate, see:
+
+- [bitcoin CHANGELOG](bitcoin/CHANGELOG.md)
+
+- [addresses CHANGELOG](addresses/CHANGELOG.md)
```
Comment (tcharding) [inline CHANGELOG.md:6]: I did not look at the rendered version but is this empty line intentional?

Comment (storopoli) [reply CHANGELOG.md:6]: Yes I was also curios about that as well

**Review by Kixunil — COMMENTED** [review]
Concept ACK, I agree with other reviewers, no other issues.

**Comment by tcharding** [comment]
Replaced by #3471, I"m pretty sure @stevenroose will be ok for me to pick this up. I did not explicitly ask him.

**Pull request closed without merging**